vector/src/main/java/im/vector/app/features/settings/devices/v2/filter/FilterDevicesUseCase.kt

```diff
@@ -31,8 +31,8 @@ class FilterDevicesUseCase @Inject constructor() {
                 .filter {
                     when (filterType) {
                         DeviceManagerFilterType.ALL_SESSIONS -> true
-                        DeviceManagerFilterType.VERIFIED -> it.cryptoDeviceInfo?.isVerified.orFalse()
-                        DeviceManagerFilterType.UNVERIFIED -> !it.cryptoDeviceInfo?.isVerified.orFalse()
+                        DeviceManagerFilterType.VERIFIED -> it.cryptoDeviceInfo?.trustLevel?.isCrossSigningVerified().orFalse()
```

Comment:
Same remark as a previous one, maybe we should rely on the `RoomEncryptionTrustLevel` field.

Reply:
I will create a dedicated PR to refactor after talking to crypto team.